module/ElasticSearch/src/ElasticSearch/VuFind/RecordDriver/ESBibliographicResource.php

```diff
@@ -38,6 +38,16 @@
  */
```

Comment:
Added this implementation of the newly introduced abstract ElasticSearch::getName() method.